docs/02-dsp-ontologies/introduction.md

```diff
@@ -4,6 +4,9 @@
 -->
 
 # Introduction
+The DSP ontologies provide a generic framework for describing humanities
```

Comment:
IMO there should be a blank line after a header in markown. (just to continue with my nit-picky comments today ^^)

Reply:
All good, I'm always happy about your nit-picky comments :)